### PR TITLE
Update fields, add [WI] race fetching

### PR DESCRIPTION
**Pull request by alxmjo**
- Update fields we want to collect for races (following discussion with Alice)
- Add fetching for **WI**

### CODE REVIEW

File: dataset/races.yaml [hunk 1]
```diff
@@ -4,9 +4,60 @@
 name: races
 index: [STATE]
 
-# Positive, death, negative X Race, Enticity
+# Positive, death, negative X Race, Ethnicity
```
Comment (alxmjo) [inline dataset/races.yaml:7]: Need to add hospitalization here.

File: dataset/races/mappings.yaml [hunk 2]
```diff
@@ -17,3 +17,28 @@ NM:
   other: POS_OTHER
   white: POS_WHITE
   hispanic: POS_HISPANIC
```
Comment (alxmjo) [inline dataset/races/mappings.yaml:19]: Need to fix fields for existing states.

File: dataset/races.yaml [hunk 3]
```diff
@@ -4,9 +4,60 @@
 name: races
 index: [STATE]
 
-# Positive, death, negative X Race, Enticity
+# Positive, hospitalization, death, negative X Race, Ethnicity
```
Comment (space-buzzer) [inline dataset/races.yaml:7]: Oops, typos :(
Thanks for fixing!

File: dataset/races/urls.yaml [hunk 4]
```diff
@@ -11,3 +11,35 @@ NM:
...
+      DTH_AIAN,
+      DTH_ASN,
+      DTH_BLK,
+      DTH_WHT,
+      DTH_MLTOTH,
+      DTH_UNK,
+      DTH_E_HSP,
+      DTH_E_NHSP,
+      DTH_E_UNK',
+    where: 1=1}
```
Comment (space-buzzer) [inline dataset/races/urls.yaml:44]: I generally use the JSON notation for `params` (and not YAML's generic item per line), so it looks closer to how it'll look in a URL, and sometimes I'd just copy it directly from the URL, also takes up less lines

**Comment by space-buzzer** [comment]
@alxmjo can you rebase and squash your PR so it's a fast-forward commit on top of master?

**Pull request merged**